pkg/tools/opa/check_name.rego

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2021 Doc.ai and/or its affiliates.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+package test
+
+default check_name = false
+
+check_name {
+    input.path_segments[_].name == "test-name"
+}
```

Comment:
@Bolodya1997 Am I get it correctly that for rejecting forwarder we'll use opa policy focused on name prefix?

Reply:
It is just a policy for the unit tests.
Here is a policy for rejecting forwarder - https://github.com/networkservicemesh/deployments-k8s/blob/8838e72452853399c3336e0f878780660c0663dc/examples/use-cases/Kernel2Kernel%26Vfio2Noop/policies-configmap.yaml#L7.